src/SendGrid/BaseClient.cs

```diff
@@ -42,7 +42,7 @@ public abstract class BaseClient : ISendGridClient
         /// </summary>
         /// <param name="options">A <see cref="BaseClientOptions"/> instance that defines the configuration settings to use with the client.</param>
         /// <returns>Interface to the Twilio SendGrid REST API.</returns>
-        public BaseClient(BaseClientOptions options)
+        protected BaseClient(BaseClientOptions options)
```

Comment:
Why the change to protected?

Reply:
In C# there's no such thing as public abstract base constructors. This change just makes that fact explicit.

Reply:
Ah, missed the abstract part 👍